src/App.js

```diff
@@ -42,21 +42,21 @@ export default class App extends Component {
             <Navbar.Brand href="#home" id="title">City Explorer</Navbar.Brand>
           </Navbar>
         </Container>
-        
+
         <Container id="form">
           <FloatingLabel controlId="floatingInputGrid" label="search for a city">
             <Form.Control id="formInput" onChange={(e) => this.setState({ exploreQuery: e.target.value })} value={this.state.exploreQuery} type="city" placeholder="search for a city" />
           </FloatingLabel>
           <Button onClick={this.handleButtonClick} id="button" size="lg">Explore!</Button>
         </Container>
-      
+
         {' '}
         <Container id="map">
           {this.state.error ? <h2>not a valid location</h2>
             : this.state.location.place_id &&
           <>
-            <h2>City Name: {this.state.location.display_name}</h2> 
-            <h3>Latitude: {this.state.location.lat}.</h3> 
+            <h2>City Name: {this.state.location.display_name}</h2>
+            <h3>Latitude: {this.state.location.lat}.</h3>
             <h3>Longitude: {this.state.location.lon}. </h3>
             <Image src={this.state.image} alt="map" thumbnail></Image>
           </>}
```

Comment:
Hi Sarah this code looks good!

To refactor it a bit you could take the container component that contains the button and text input field form and separate logic into its own component.

Same idea for the container with id="map". 

Then by passing a prop with a function to modify those state values in App.js this component could be cleaned up.